pinning/draft-sheffer-tls-pinning-ticket.md

```diff
@@ -96,40 +97,6 @@ no manual management actions are required.
 
 # Introduction
 
```

Comment:
Suggest to add at the top of the Intro:

Ticket pinning is a proposed solution for the problem of mis-issued certificates, either for the global PKI or for extensive deployments of certificates within enterprises. This is an easy to implement and deploy solution, reusing	
some of the ideas behind TLS session resumption.